src/main/java/com/google/api/generator/debug/CodeGeneratorRequestDumper.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.debug;
+
+import com.google.api.generator.Main;
+import com.google.protobuf.ExtensionRegistry;
+import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest;
+import com.google.protobuf.compiler.PluginProtos.CodeGeneratorResponse;
+import java.io.IOException;
+
+public class CodeGeneratorRequestDumper {
```

Comment:
@miraleung (from https://github.com/googleapis/gapic-generator-java/pull/767): If we're using `Main` in the class names, could we append it here for consistency? Alternatively, remove `Main` from the other debugger's main class's name.

Reply:
Those are different things basically. The MainFromFile is literally another variation of the Main class. This, on the other hand, is a very different class, doing different thing. That is why they are named differently. And strictly speaking, this file is really very-very non-main =) Either way, if you have preferrence how to name these classes, please let me know and I'll rename them. For now, those were the best names I could come up with.

Reply:
MainStandalone could be something like `CodeGeneratorRequestFileToGapicMain`?

Reply:
Renamed